src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -317,9 +314,10 @@ bool WriteLog<I>::initialize_pool(Context *on_finish, pwl::DeferredContexts &lat
     } TX_ONABORT {
       this->m_total_log_entries = 0;
       this->m_free_log_entries = 0;
-      lderr(cct) << "failed to initialize pool (" << this->m_log_pool_name << ")" << dendl;
-      on_finish->complete(-pmemobj_tx_errno());
-      return false;
+      lderr(cct) << "Failed to initialize pool: " << this->m_log_pool_name
+                 << ". Pmemobj TX errno: " << pmemobj_tx_errno() << dendl;
```

Comment:
Just noticed after the second review, that there is `pmemobj_errormsg()`. So just wonder, could it be used here or if not may be there is something similar for  `pmemobj_tx_errno()`? If not, please just ignore. It looks good enough.

Reply:
I don't know if transaction errors will be the same with pmemobj_errormsg(). But I don't think this is very important. Later, it is planned to change `libpmemobj` to `libpmem`. This PR I had @ you. And the transaction will be removed. Now let's keep it as it is.